<?xml version="1.0" encoding="UTF-8"?><testrun name="Hooks" project="CTAppium" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="stepsDefinitions.Hooks" time="5.301">
    <testcase name="sampleTest" classname="stepsDefinitions.Hooks" time="5.301">
      <error>org.openqa.selenium.WebDriverException: Connection refused: connect
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SAO-6X7S9R2', ip: '10.212.128.36', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: driver.version: AndroidDriver&#13;
	at io.appium.java_client.remote.AppiumCommandExecutor.lambda$5(AppiumCommandExecutor.java:251)&#13;
	at java.util.Optional.orElseGet(Unknown Source)&#13;
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)&#13;
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)&#13;
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)&#13;
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)&#13;
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)&#13;
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)&#13;
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:88)&#13;
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:98)&#13;
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:94)&#13;
	at core.DriverFactory.createDriver(DriverFactory.java:24)&#13;
	at core.DriverFactory.getDriver(DriverFactory.java:32)&#13;
	at stepsDefinitions.Hooks.startAppium(Hooks.java:16)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
Caused by: java.net.ConnectException: Connection refused: connect&#13;
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.PlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.SocksSocketImpl.connect(Unknown Source)&#13;
	at java.net.Socket.connect(Unknown Source)&#13;
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)&#13;
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)&#13;
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)&#13;
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)&#13;
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)&#13;
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)&#13;
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)&#13;
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)&#13;
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)&#13;
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)&#13;
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)&#13;
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)&#13;
	at okhttp3.RealCall.execute(RealCall.java:77)&#13;
	at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)&#13;
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)&#13;
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)&#13;
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)&#13;
	... 38 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
